src/cdomains/intDomain.ml

```diff
@@ -15,6 +15,59 @@ exception Unknown
 exception Error
 exception ArithmeticOnIntegerBot of string
 
+
+
+
+(** Define records that hold mutable variables representing different Configuration values.
+  * These values are used to keep track of whether or not the corresponding Config values are en-/disabled  *)
+  type ana_int_config_values = {
+    mutable interval_threshold_widening : float_precision option;
+    mutable interval_narrow_by_meet : float_precision option;
+    mutable def_exc_widen_by_join : float_precision option;
```

Comment:
```suggestion
    mutable interval_threshold_widening : bool option;
    mutable interval_narrow_by_meet : bool option;
    mutable def_exc_widen_by_join : bool option;
```

Reply:
Thanks for the tip! Should be fixed in https://github.com/goblint/analyzer/pull/943/commits/f5859c39a057ab937860b07c87eceaa217f4b49c